source/style.css

```
@@ -61,7 +61,10 @@ button:hover,
  */
 .header {
 	display: flex;
-	gap: 0.5em;
+	position: sticky;
+	z-index: 100;
+	top: var(--margin);
+	outline: solid var(--margin) #2b2c2c;
```

I think this fixes support for light mode too. Can you verify? Make send a new gif

```suggestion
	outline: solid var(--margin) light-dark(#fff, #292a2d);
```